### PR TITLE
use github actions to build injector image

### PR DESCRIPTION
**Pull request by WanzenBug**
Tested here: https://github.com/WanzenBug/piraeus/runs/881227632
(fails because registry user is not configured)

### CODE REVIEW

**Comment by WanzenBug** [comment]
Forgot to put `v` in front of the docker tags... updated action log: https://github.com/WanzenBug/piraeus/runs/881272016

**Comment by WanzenBug** [comment]
Also forgot to update the makefile... brb

**Comment by rck** [comment]
I think it misses the `--build-arg` bits in the `Makefile`, if for whatever reason one builds it manually. This is one reason I prefer one source of truth, which in this case would be the `Makefile`

File: dockerfiles/drbd-driver-loader/Makefile [hunk 1]
```diff
@@ -14,7 +14,7 @@ all: update upload
 update:
 	for f in $(DF); do \
 		pd=$(PROJECT)-$$(echo $$f | sed 's/^Dockerfile\.//'); \
-		docker build --no-cache=$(NOCACHE) -f $$f -t $$pd:$(TAG) . ; \
+		docker build --build-arg DRBD_VERSION=$(shell cat DRBD_VERSION) --no-cache=$(NOCACHE) -f $$f -t $$pd:$(TAG) . ; \
```
Comment (rck) [inline dockerfiles/drbd-driver-loader/Makefile:17]: Wouldn't that be `DRBD_VERSION="$(TAG)"`

Comment (rck) [inline dockerfiles/drbd-driver-loader/Makefile:17]: hm, these `cat`s are pretty fragile, usually I'm not a big fan of them. In my experience having proper variables in a file like `DRBD_VERSION='1.2.3'` and sourcing this file in the shell like `source env; echo "$DRBD_VERSION"` is ways more robust.
`x=$(cat <(echo -e "\n\nbroken")); echo "1${x}2"`

**Comment by WanzenBug** [comment]
@rck please re-review. Basically moved everything into the makefile. The github workflow also just uses the makefile.

**Comment by rck** [comment]
jop, I like that version, thanks for adapting.

**Pull request merged**